### PR TITLE
Creation Entities

### PR DESCRIPTION
**Pull request by AngheloZambrana**
**Description:**
The Entities folder was created, in which the classes that represent the tables we have in our database were created.
[company.json](https://github.com/user-attachments/files/17477852/company.json)
[product.json](https://github.com/user-attachments/files/17477842/product.json)
[subsidiary.json](https://github.com/user-attachments/files/17477843/subsidiary.json)

Task: https://app.clickup.com/t/868accf5b
Reviewers: @JCarlosHidalgoSosa @OropezaHugo

### CODE REVIEW

File: apps/ProductManagement/ProductManagement/Entities/Company.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    public List<Guid> Subsidiaries { get; set; }
+    public string Partner { get; set; }
+    public List<Guid> Products { get; set; }
+
+    public Company()
+    {
+        Subsidiaries = new List<Guid>();
+        Products = new List<Guid>();
+    }
+}
```
Comment (OropezaHugo) [inline apps/ProductManagement/ProductManagement/Entities/Company.cs]: add endline

File: apps/ProductManagement/ProductManagement/Entities/Product.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+namespace ProductManagement.Entities;
+
+public class Product
+{
+    public Guid ProductID { get; set; }
+    public string Name { get; set; }
+    public List<string> Categories { get; set; }
+}
```
Comment (OropezaHugo) [inline apps/ProductManagement/ProductManagement/Entities/Product.cs]: add endline

File: apps/ProductManagement/ProductManagement/Entities/Subsidiary.cs [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    public double Lat { get; set; }
+    public double Lng { get; set; }
+}
+
+public class Stock
+{
+    public Guid StockID { get; set; }
+    public Guid ProductId { get; set; }
+    public int Quantity { get; set; }
+}
```
Comment (OropezaHugo) [inline apps/ProductManagement/ProductManagement/Entities/Subsidiary.cs]: add endline

File: apps/ProductManagement/ProductManagement/Entities/Company.cs (hunk 1 above)
Comment (AngheloZambrana) [reply apps/ProductManagement/ProductManagement/Entities/Company.cs]: Fixed in the last commit

File: apps/ProductManagement/ProductManagement/Entities/Product.cs (hunk 2 above)
Comment (AngheloZambrana) [reply apps/ProductManagement/ProductManagement/Entities/Product.cs]: Fixed in the last commit

File: apps/ProductManagement/ProductManagement/Entities/Subsidiary.cs (hunk 3 above)
Comment (AngheloZambrana) [reply apps/ProductManagement/ProductManagement/Entities/Subsidiary.cs]: Fixed in the last commit

**Comment by JCarlosHidalgoSosa** [comment]
Screenshot from 2024-10-23 09-44-54
Solve warnings.

**Review by JCarlosHidalgoSosa — APPROVED** [review]

**Review by OropezaHugo — APPROVED** [review]

**Pull request merged**